test/pages.js

```diff
@@ -421,6 +421,16 @@ describe('Pages', function() {
     assert.strictEqual(page.rank, 1);
   });
 
+  it('is not able to move a page under itself', async function() {
+    await assert.rejects(
```

Comment:
Love that `assert.rejects` need to use it more, it's very awkward to write this kind of test correctly otherwise, often done wrong